mtp_prisoner_location_admin/templates/prisoner_location_admin/location_file_upload.html

```diff
@@ -1,9 +1,17 @@
 {% extends "base.html" %}
+{% load i18n staticfiles %}
 
 {% block content %}
-<form action="{% url "location_file_upload" %}" method="post" enctype="multipart/form-data">
+<h1>{% trans "Upload Location File" %}</h1>
+<form action="{% url "location_file_upload" %}"
+			method="post"
+			enctype="multipart/form-data"
+			class="js-uploadSubmit">
 	{% csrf_token %}
+	<label for="id_location_file">{% trans "Choose file to upload" %}</label>
+	<br/>
```

Comment:
I guess my point was more “can't we just use `display: block` and some margin on the label?”.